frontend/components/institutions/Institution.tsx

```diff
@@ -199,6 +203,16 @@ export default function Institution({ institutionId }: InstitutionProps) {
                   />,
                 ]
               : []
+          )
+          .concat(
+            institution.canCurrentUserDeleteNode
```

Comment:
`canCurrentUserDeleteNode` is not what you want here. What you want is something along the lines `canCurrentUserChangeOperatingStateOfNode`. This needs to be implemented then along the lines of `canCurrentUserDeleteNode`. If you cannot find in the code please tell me and I'll give you further support.